docs/CloudPermissions.md

```diff
@@ -0,0 +1,58 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
+
+This guide explains how to assign required permissions for the users for deploying Scalar DL on Azure.
+
+From the security perspective, it is better to give users limited privileges to protect the environment from unwanted activities.
+
+## Azure
+
+The following JSON is a custom role that allows users to manage Azure cloud resources for Scalar DL. These are the minimum permissions for creating a Scalar DL environment on the Azure cloud.
```

Comment:
Ditto.

Reply:
The permissions are for creating and managing all the resources that are needed for Scalar DL deployment.
Modified the sentence with the command line tools also can be used with these permissions.